extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfLogHandlerRecorder.java

```diff
@@ -26,6 +28,33 @@
         handler.setTimestampPattern(config.timestampPattern);
         handler.setHost(config.host);
         handler.setPort(config.port);
+        handler.setLevel(config.level);
+
+        // handle additional fields
+        if (!config.additionalField.isEmpty()) {
+            StringBuilder additionalFieldsValue = new StringBuilder();
+            StringBuilder additionalFieldsType = new StringBuilder();
+            for (Map.Entry<String, AdditionalFieldConfig> additionalField : config.additionalField.entrySet()) {
+                if (additionalFieldsValue.length() > 0) {
+                    additionalFieldsValue.append(',');
+                }
+                additionalFieldsValue.append(additionalField.getKey()).append('=').append(additionalField.getValue().value);
```

Comment:
There's no escaping needed?

Reply:
In all examples that I saw, there was no escaping.
I don't know what will happens if a user put a special character in an additional field.
As it comes from a configuration file, there is no security risk so I propose to let it this way